controller/iteration_blackbox_test.go

```diff
@@ -77,13 +77,10 @@ func (rest *TestIterationREST) TestCreateChildIteration() {
 
 	rest.T().Run("success - create child iteration", func(t *testing.T) {
 		fxt := tf.NewTestFixture(t, rest.DB,
-			tf.Spaces(1, tf.SetSpaceIDsFromString([]string{"7fa96f9a-1900-47ee-b375-1c2eee31fbbb"})),
-			tf.Identities(1, tf.SetIdentityIDsFromString([]string{"c21f96dd-493e-4555-9d32-9284b3883724"})),
-			tf.Areas(1, tf.SetAreaIDsFromString([]string{"030b0865-4215-487b-a52c-db554fcab865"})),
+			tf.Identities(1),
+			tf.Areas(1),
 			tf.Iterations(2,
 				tf.SetIterationNames([]string{"root iteration", "child iteration"}),
```

Comment:
Hi @pranavgore09 . Notice that you now can just pass the strings to this function instead of passing an array ;)

Reply:
Yeah, it failed to compile and I fixed it while merging master